### PR TITLE
Fix nullable boolean

### PR DESCRIPTION
**Pull request by paulboudewijn**
Solution regarding discussion #8488 (Interface doesn't support nullable boolean)

### CODE REVIEW

File: app/src/interfaces/boolean/boolean.vue [hunk 1]
```diff
@@ -45,6 +49,30 @@ export default defineComponent({
 			default: '#00C897',
 		},
 	},
+	methods: {
```
Comment (azrikahar) [inline app/src/interfaces/boolean/boolean.vue:52]: Would you be ok to change this to Composition API? Just for consistency sake 👍 

The `input.vue` should be a good reference for this.

Comment (paulboudewijn) [reply app/src/interfaces/boolean/boolean.vue]: Done!

Comment (azrikahar) [reply app/src/interfaces/boolean/boolean.vue]: Thanks!

**Comment by paulboudewijn** [comment]
Hi @rijkvanzanten,
Using a selectbox with the values true, false and null seems to work more intuitively then a tri-state checkbox. 
I've removed all changes to the checkbox interface, so only the fix regarding displaying the raw value of a (nullable) bool remains.
Can this be part of v9.0.0-rc.101 ?

**Review by rijkvanzanten — APPROVED** [review]

**Comment by rijkvanzanten** [comment]
Zekersteweten paul 💯

**Pull request merged**